app/routers/sessions.py

```diff
@@ -47,7 +51,10 @@ async def create_session(session: Session):
 
         # restore the answers from the previous sessions
         session_answers = list(
-            client.quiz.session_answers.find({"session_id": previous_session["_id"]})
+            client.quiz.session_answers.find(
+                {"session_id": previous_session["_id"]},
+                sort=[("_id", pymongo.ASCENDING)],
```

Comment:
Why sort again? We will match question ids to show previously answered questions, no?

Reply:
please read my reply to the comment above - it is because we are storing the first few session answers in the session object itself, in which case, the session answer objects have to be stored in a fixed order.